tasks/styles.js

```diff
@@ -45,6 +46,9 @@ module.exports = (gulp, $, pkg) => {
         }),
         outputStyle: options.outputStyle
       }).on('error', reportError))
+      .pipe(function() {
```

Comment:
@juliquiron We should probably wrap this with `$.if(options.production)` similar to other lines in this file. I guess we don't need hashes during development.

Reply:
I would recommend to enable the hashes at all times, as Expires/Cache-Control might get enabled permanently in the future.